python/lsst/meas/algorithms/astrometrySourceSelector.py

```diff
@@ -0,0 +1,121 @@
+# See COPYRIGHT file at the top of the source tree.
```

Comment:
I'm taking this from jointcal, where I was told to do it that way (THE FUTURE!). I'll check with SQuaRE.